docs/topics/proc_deploying-the-secured-booster-using-the-oc-cli-client.adoc

```diff
@@ -0,0 +1,81 @@
+
+[#deploying-the-secured-booster-using-the-oc-cli-client_{context}]
+= Deploying the {name-mission-secured} booster using the `oc` CLI client
+
+.Prerequisites
+
+* The booster application created using the {launcher} application.
+ifndef::parameter-ocp[For more information, see xref:creating-the-secured-booster-using-launcher_{context}[].]
+
+ifndef::parameter-ocp[* Your {launcher} URL.]
+* The `oc` client authenticated. For more information, see xref:authenticating-the-oc-cli-client-to-access-launcher_{context}[].
+
+.Procedure
+
+. Create a new project.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc new-project {project-name}
+----
+
+. Navigate to the root directory of your booster.
+
+. Deploy the {RHSSO} server using the `service.sso.yaml` file from your booster ZIP file:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc create -f service.sso.yaml
+----
+
+. Get the public address of the {RHSSO} server:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get route secure-sso -o jsonpath='{"https://"}{.spec.host}{"/auth\n"}'
+----
+
+ifndef::http-api-nodejs[]
+. Use Maven to start the deployment to {OpenShiftLocal}.
++
+--
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:deploy -Popenshift -DskipTests -DSSO_AUTH_SERVER_URL=<SSO_AUTH_SERVER_URL>
```

Comment:
I know this works with Swarm, but someone from Vert.x and Spring Boot should review as well.